2.4/.sti/bin/assemble

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+echo "---> Installing application source"
```

Comment:
add " ..." suffix (we agreed to change it in all images, so I will be now enforcing it in all pull requests ;-)